.gitignore

```diff
@@ -7,4 +7,6 @@ site_data.csv
 *.svg
```

Comment:
One thing still needs to be squared away: all the .DS_Store files and the .Rapp.history need to be ignored